Example/AblyChatExample/Mocks/MockSubscriptionStorage.swift

```diff
@@ -1,4 +1,5 @@
-import Foundation
+import Ably
+@testable import AblyChat
 
 // This is copied from ably-chat’s internal class `SubscriptionStorage`.
```

Comment:
This is no longer used.

Reply:
`SubscriptionStorage` now stores `Subscription` (callback variant) and is used in `DefaultRoomLifecycleManager` for debug methods in tests.